.github/workflows/actions.yml

```diff
@@ -11,16 +11,16 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Setup Node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v4
         with:
-          node-version: 12
+          node-version: 18

```

Comment:
This is required because this is the node version that supports esm modules. Without this, the build step fails.